### ISSUE TITLE
The installer fails when trying to install vc redist

### ISSUE THREAD

**Issue by moozzyk** [rjpcomputing/luaforwindows#114]
Run the installer from here: https://github.com/rjpcomputing/luaforwindows/releases/tag/v5.1.5-51 using "Run As Administrator". 

![image](https://cloud.githubusercontent.com/assets/1438884/20028744/1faa7f02-a2f7-11e6-99ac-abf74f8e34e3.png)

Environment: Windows 10, processor: x64.

(if this is reall vs 2005 redist it's quite old)

**Comment by moozzyk** [comment]
The installer failed but REPL seems to be working.

**Comment by rjpcomputing** [comment]
Yes it is built using visual studio 2005. That is the only option right now
because Lua for Windows is only a binary distribution. It doesn't include
modules it builds itself. And all of them were built with vs2005. There
will be an effort soon to change to Lua 5.3 and LuaRocks. This way it
becomes less modules but users can add more thru LuaRocks.

Hope this explanation helps.
--
Regards,
Ryan

**Comment by moozzyk** [comment]
That's all right but still the failure is confusing (especially that things appear to work)

**Comment by rjpcomputing** [comment]
@moozzyk,

I completely agree. I wonder if the VC2005 redist doesn't work in Windows 10. Maybe they built it in now. I have no real testing in Windows 10 yet.

Also could you try to just run this VS2005 redist (https://github.com/rjpcomputing/luaforwindows/releases/download/vc-runtime/vcredist_x86.4053.exe). It is the one that is used during LfW installer.

**Comment by dbsxdbsx** [comment]
I have the same problem under win7 64, but it still seems work. but I do not know how to let lua  work under CMD.

**Comment by idbrii** [comment]
I encountered the same issue (but wasn't running as admin) and found that the vcredist file (both in the same directory as LuaForWindows_v5.1.5-51.exe and in install\support\ were 0 bytes. Replacing them with the exe @rjpcomputing linked fixed the issue (Win10 Pro x64).

Maybe the installer is failing to download properly?
